src/Microsoft.Net.Http.Headers/EntityTagHeaderValue.cs

```diff
@@ -112,6 +112,24 @@ public override int GetHashCode()
             return _tag.GetHashCode() ^ _isWeak.GetHashCode();
         }
 
+        public static bool CompareStrongly(EntityTagHeaderValue left, EntityTagHeaderValue right)
```

Comment:
Do we need (EntityTagHeaderValue.CompareStrongly(EntityTagHeaderValue other)` as well?